spec/back_end/features/list_spaces_spec.rb

```diff
@@ -0,0 +1,20 @@
+feature 'list spaces' do
+  scenario 'an authenticated user can list a space' do
+    sign_up
+    sign_in
```

Comment:
we have redefined the <code>sign_up</code> and <code>sign_in</code> web_helpers; this may change these tests